### ISSUE TITLE
Review and refactor theming topics

### ISSUE THREAD

**Issue by gregvanl** [microsoft/vscode-docs#3299]
Currently there are 4 theme related topics:
/docs/getstarted/themes.md
/api/references/color-theme.md
/api/extension-capabilities/color-theme.md
/api/extension-guide/color-theme.md

Users can land on any one of them from search and are confused. From verbatim, most users just want to adjust their current theme.
We should review and refactor these topics and give clear direction depending on that the reader wants to do.

**Comment by httpsterio** [comment]
Hey, as a theme developer I agree that the guidelines for theming are a bit confusing as the information seems to be scattered here and there. 

I have a few issues related to the documentation of themes currently.

1. As stated in this issue, the information is scattered and there's honestly too much of it. It needs a rewrite or at least it needs to be recompiled and reorganized again.

2. There is no easy way to discern what the names of each element of the UI is called. If I know that the Command Palette is called just that, I can control+f and see that it's grouped under _quickInputTitle_  in the documentation. But what about the rest of the UI that doesn't have easy names like that? There are some images and gifs in the documentation and it's a big help, but there's not enough of those.

3. Changelog for themes. I just got notified by a user that my theme has a broken color. I have no idea which update broke it, I don't know which tokens have been added/deprecated/removed and there's really not any way for me to keep up with the changes. 

I would really like a way to either pick a theme and have something / some place shouting at me that X tags are not included in the theme or this has been released on date Y and VS-Code has brought Z things out since that themes need to factor in.

Either that, or just a simple changelog link on the relevant page that explains all of the edits on each version upgrade that relates to themes.
